recipes/osgearth/all/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+cmake_minimum_required(VERSION 3.1)
+project(cmake_wrapper)
+
+include(conanbuildinfo.cmake)
+conan_basic_setup()
+
+if(WIN32)
+    add_compile_definitions(NOGDI)
+endif()
+
+# find_package and set the expected variables
+
+# OSG
+find_package(OpenSceneGraph REQUIRED)
+
+set(OSG_PLUGINS "osgPlugins-${OPENSCENEGRAPH_VERSION}"  CACHE STRING "" FORCE)
+set(OSG_LIBRARY ${OPENSCENEGRAPH_LIBRARIES})
+set(OSG_FOUND ${OPENSCENEGRAPH_FOUND})
+set(OSG_INCLUDE_DIRS ${OPENSCENEGRAPH_INCLUDE_DIRS})
+set(OSG_INCLUDE_DIR "${OPENSCENEGRAPH_INCLUDE_DIRS}")
+set(OSG_GEN_INCLUDE_DIR "${OPENSCENEGRAPH_INCLUDE_DIRS}")
+
+# gdal
+find_package(GDAL REQUIRED)
+set(GDAL_LIBRARY ${GDAL_LIBRARIES})
+
+# curl
+find_package(CURL REQUIRED)
```

Comment:
I am a little surprised to see this here.
https://github.com/gwaldron/osgearth/blob/master/CMakeLists.txt
Has this already. I am not familiar with the project but some of this logic seems a tad extra.

Is everything required?

Reply:
osgearth has custom cmake find modules which unfortunately set different variables than the ones generated by conan (most notably *_LIBRARY instead of *_LIBRARIES, but also different names). These are then used in multiple places throughout, so I opted for setting these non-standard variables in the wrapper.

I agree though, that since I do a patch anyway, the wrapper could probably eliminated. Do you think it is necessary?

Reply:
The wrapper is needed and I suspect Conan V2 will have an impact...

Let's improve the comments!